heat/preprocessing/preprocessing.py

```diff
@@ -129,10 +129,10 @@ def transform(self, X: ht.DNDarray) -> ht.DNDarray:
         _has_n_features(self.mean_, X)
         if self.copy:
             return (X - self.mean_) * self.scale_
-        # else in place:
```

Comment:
The original version was correct due to the return in the if-clause (same at further places here)

Reply:
Got it, thank you.